bin/v-add-user-ssh-jail

```diff
@@ -0,0 +1,94 @@
+#!/bin/bash
+# info: add user ssh jail
+# options: USER [RESTART]
+#
+# example: v-add-user-ssh-jail admin
+#
+# This function enables ssh jailed environment
+
+#----------------------------------------------------------#
+#                Variables & Functions                     #
+#----------------------------------------------------------#
+
+# Argument definition
+user=$1
+restart=$2
+
+# Includes
+# shellcheck source=/etc/hestiacp/hestia.conf
+source /etc/hestiacp/hestia.conf
+# shellcheck source=/usr/local/hestia/func/main.sh
+source $HESTIA/func/main.sh
+# load config file
+source_conf "$HESTIA/conf/hestia.conf"
+
+chroot="/srv/jail/$user"
+
+#----------------------------------------------------------#
+#                    Verifications                         #
+#----------------------------------------------------------#
+
+# Checking if jailkit is installed
+if [ ! -x /sbin/jk_init ] ; then
+    echo "Error: Jailkit isn't installed"
```

Comment:
If jail kit doens't exists we should build packages for it and ship it

Reply:
But existing installs will break see:

https://drone.hestiacp.com/hestiacp/hestiacp/3535/1/5

We can't have it ..

Reply:
Ah ok, it doesn't like loud errors. I will remove the error messages. Jailkit not being installed should be an okay state so users have the option for non-jailed bash.

If you have a suggestion on how to support jailed with non-jailed bash in one installation I'm all ears. I couldn't think of one ^^°.